x-pack/qa/saml-idp-tests/src/test/java/org/elasticsearch/xpack/security/authc/saml/SamlAuthenticationIT.java

```diff
@@ -240,7 +240,7 @@ public void setupNativeUser() throws IOException {
      * <li>Uses that token to verify the user details</li>
      * </ol>
      */
-    @AwaitsFix(bugUrl = "https://github.com/elastic/elasticsearch/issues/44410")
+    //@AwaitsFix(bugUrl = "https://github.com/elastic/elasticsearch/issues/44410")
```

Comment:
Can we just remove these annotations instead of commenting them out?

Reply:
will do before merging, I wanted to help us((me) not forget that it depends on #44410